src/Statement/Grouper/StatementGrouper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Statement\Grouper;
+
+use Wikibase\DataModel\Statement\StatementList;
+
+/**
+ * @since 3.2
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+interface StatementGrouper {
+
+	/**
+	 * @param StatementList $statements
+	 *
+	 * @return StatementList[] An associative array, mapping statement group identifiers to
```

Comment:
The goal of these questions is for me to understand something which I currently do not. "Is there something these identifiers have in common" is a specific question.

Interfaces are much closer related to their users than to their implementations. The question I'm asking is about intend and cannot be answered just by looking at the implementations.